core/types/tx_rip7560.go

```diff
@@ -187,19 +190,32 @@ func (tx *Rip7560AccountAbstractionTx) AbiEncode() ([]byte, error) {
 	args := abi.Arguments{
 		{Type: structThing, Name: "param_one"},
 	}
+
+	paymaster := tx.Paymaster
+	if paymaster == nil {
+		paymaster = &common.Address{}
+	}
+	deployer := tx.Deployer
+	if deployer == nil {
+		deployer = &common.Address{}
+	}
```

Comment:
@shahafn @drortirosh I think we need to decide whether we treat zero address as a `nil` or if we want to actually prevent that. What do you think?

Reply:
Currently it crashes when abi encoding it with nil, so we need to put zero values there